database/youtube/web-development.json

```diff
@@ -61,5 +61,22 @@
         "category": "youtube",
         "subcategory": "web-development",
         "language": "hindi"
+    },
+    {
+        "name": "DesignCourse",
+        "description": "This channel will provide and take you from zero to a fullstack developer.you will learn UI/UX, Frontend Dev",
+        "url": "https://www.youtube.com/@DesignCourse/featured",
+        "category": "youtube",
+        "subcategory": "web-development",
+        "language": "English"
+    },
+    {
+        "name": "Bro Code",
+        "description": "This channel is dedicated to providing free education on coding bootcamp, and whatever coding gurus sell",
```

Comment:
Great start @darkweb907! 👍🏾 However, the word “bootcamp” implies that this YouTuber sells free education about coding bootcamps instead of coding lessons.